fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -87,14 +87,12 @@ def valid?(value)
 
     # Returns an updated value type (if necessary)
     def auto_convert_value(value)
-      # Weird because of https://stackoverflow.com/questions/9537895/using-a-class-object-in-case-statement
-      case
-      when data_type == Array
+      if data_type == Integer
+        return value.to_i unless value.nil? || value.to_i.to_s != value.to_s
```

Comment:
We don't want to check for the value being greater than 0 because 0 could be a valid input depending on the action. We could refactor to break it into another line or two, but I believe this is the logic we need to use here.